docs/guide.rst

```diff
@@ -855,6 +855,36 @@ that repository.
         how: fmf
 
 
+.. _adjust-metadata:
+
+Adjust Metadata
+------------------------------------------------------------------
+
+
+Sometimes metadata needs to be adjusted based on the context.
+For example, the user might want to enable a test only for a
+specific architecture or skip a plan when running in a container.
+The core attribute :ref:`/spec/core/adjust` provides a flexible
```

Comment:
I'm confused about what this section tries to document, since it's covered in the `spec/core/adjust`. If it's meant to be as a quick introduction, then could it be more brief, maybe focusing on the example only?